src/Context.php

```diff
@@ -16,6 +16,9 @@ class Context
     /** @var string */
     private $signingKeyId;
 
+    /** @var AlgorithmInterface */
+    public $algorithm;
```

Comment:
private

Reply:
Resolved in 1143d2a9e043b91846ad760f21c3b15f1fc85750